Octokit.Generators/Octokit.Generators.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>A set of code generators for Octokit.NET backed by the GitHub REST API Open API descriptions</Description>
+    <AssemblyTitle>Octokit.Generators</AssemblyTitle>
+    <Authors>GitHub</Authors>
+    <Version>0.0.0-dev</Version>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <AssemblyName>Octokit.Generators</AssemblyName>
+    <PackageId>Octokit.Generators</PackageId>
+    <DebugType>embedded</DebugType>
+    <RepositoryUrl>https://github.com/octokit/octokit.net</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/octokit/octokit.net</PackageProjectUrl>
```

Comment:
Does this project need package details as I assume it’s not getting packed up?

Reply:
So this is me thinking forward... I am still not sold as this being part of the .NET SDK itself, so I added that in there as a placeholder just in case we want to ship this as a package... I could remove it or leave it - I can see how it's a little extra cruft.

Reply:
Ahh yep sorry, didn’t think of that! Definitely a good thing to have even if it’s just there to make life easier in the future. ❤️